python/ray/train/v2/_internal/execution/worker_group/state.py

```diff
@@ -133,3 +134,34 @@ def _shutdown_sync_actor(sync_actor: SynchronizationActor):
 
 def _shutdown_placement_group(placement_group: PlacementGroup):
     remove_placement_group(placement_group)
+
+
+@dataclass(frozen=True)
+class WorkerGroupResizeStatus(PolicyHandledStatus):
```

Comment:
What about naming this `WorkerGroupSchedulingStatus`? And `WorkerGroupPollStatus` can be renamed to `WorkerGroupRunningStatus`.

Then, these status names match their respective controller states.